test/pizza-guy.spec.js

```diff
@@ -64,4 +79,40 @@ describe('pizza-guy', function() {
       expect(pizzaGuy.deliver([]).start()).toBe(undefined);
     });
   });
+
+  describe('physical files', function() {
+    it('should call onComplete after all the images where downloaded', function(done) {
+      this.timeout(4000);
```

Comment:
So basically what I don't like about timeouts in general is that you say for example `this.timeout(4000)` but you don't have a strong reason to use 4000 instead of 4100, 4200, 8000, 16000...100000 [ _magic numbers_ ]

But yeah, if you don't mind leaving the timeout call, I'm ok with it + mocha supports it https://mochajs.org/#timeouts